spec/features/staff/users_can_create_a_transaction_spec.rb

```diff
@@ -257,6 +257,20 @@
         expect(page).to have_content t("activerecord.errors.models.transaction.attributes.date.blank")
       end
     end
+
+    scenario "they can cancel their transaction" do
```

Comment:
How about something like "they can choose not to submit a transaction"? This reads like 'cancel' is a specific action.